google-spreadsheet-config.json

```diff
@@ -0,0 +1,5 @@
+{
+  "spreadsheetId": "1QlEqI-HMumoSDp6HO9tSkLWluoioIZz4U1q1z0DAFD0",
```

Comment:
Yes, it is the id of a spreadsheet I created for testing purposes. I should change this to `"spreadsheetId": "<you spreadsheet id here>"`